src/validation.h

```diff
@@ -427,20 +427,16 @@ class BlockManager
 
     std::unique_ptr<CBlockTreeDB> m_block_tree_db GUARDED_BY(::cs_main);
 
-    bool LoadBlockIndexDB(std::set<CBlockIndex*, CBlockIndexWorkComparator>& setBlockIndexCandidates) EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
+    bool LoadBlockIndexDB(ChainstateManager& chainman) EXCLUSIVE_LOCKS_REQUIRED(::cs_main);
```

Comment:
36e2bc4275fdb0d72390215fcea4d499f8049701: is it possible to spit this out as a refactor commit? IIUC `setBlockIndexCandidates` here is always just  `ActiveChainstate().setBlockIndexCandidates`.

Reply:
It's possible, but I'm not going to do it because it creates too much intermediate churn (updating code that is changed anyway in the subsequent commit).